rootfs/ingest/python/qserv/queue.py

```diff
@@ -217,27 +213,23 @@ def _acquire_mutex(self) -> None:
                 wait_sec = util.increase_wait_time(wait_sec)
 
     def _release_mutex(self) -> None:
-        release_mutex_query = update(self.mutex).values(
-            pod=None,
-            latest_move=datetime.datetime.now()
-        )
+        release_mutex_query = update(self.mutex).values(pod=None, latest_move=datetime.datetime.now())
         release_mutex_query = release_mutex_query.where(self.mutex.c.pod == self.pod)
         self._safe_execute(release_mutex_query, _MAX_RETRY_ATTEMPTS)
 
     def init_mutex(self) -> None:
-        """Initialize mutex in queue database
-           Queue database has a table `mutex` which contain only one row. This row is used to enable
-           only one pod to lock contribution file in queue at a time and need to be initialized
-           at ingest startup.
+        """Initialize mutex in queue database Queue database has a table
+        `mutex` which contain only one row.
```

Comment:
Needs double backticks around `mutex` here, as it does not refer to a type or variable.